integration-cli/docker_api_containers_test.go

```diff
@@ -552,7 +552,7 @@ func (s *DockerSuite) TestContainerApiCommit(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-	if cmd != "{[/bin/sh -c touch /test]}" {
```

Comment:
Unfortunately runconfig is both an internal and an external data structure. (and image config).
It's used as the API types for creating new containers as well as stored on disk (but only after it's mangled a bit).
It's really a nasty mess :(